javascript/first-js-oop-project.md

```diff
@@ -0,0 +1,138 @@
+# Mini Project: Object Oriented Programming in JS
+
+### Projected Time
+
+Total: 65 min- 80min
+- Independent Practice: 15 min
+- Materials: 25 min
+- Lesson: 30 min
+
+### Prerequisites
+
+- [Java script lesson 7](../javascript/javascript-7-oop.md)
+
+
+### Objectives
+
+- To use the new JS object-oriented programming skills.
+- To create a few event objects and push them to an array.
+- Use jQuery to display the list of events to an html page.
+
+### Specific Things To Teach
+
+- Creating event array.
+- Iterating through the array.
+- Displaying array items in HTML page.
+
+### Materials
+
+- [Documentation on each statement of jQuery](http://api.jquery.com/jquery.each/)
+- [Arrays in javascript](https://www.w3schools.com/js/js_arrays.asp)
+- [.html function of jQuery](https://www.w3schools.com/jquery/html_html.asp)
+
+### Lesson
+
+- The below example is the just a part of a mini project that will help you to learn basic concept about arrays in javascript and iterating through the arrays. The above code will allow you to learn about displaying the array objects in the web pages.
+- Once you are done with the code, run the index.html file in your browser to check the result.
+- Keep on refreshing the browser to keep track of the changes you make everytime.
+
+- Create a js file ***events.js***. Starting with the project, first we need to create an empty array.
+```javascript
+  class events {
+  constructor(name, description) {
+    this.name = name;
+    this.description = description;
+  }
+  let events = new events("KLOS Golden Gala", "An evening with hollywood vampires");
+```
+
+- Begining with pushing single items to the array, you need to use ***.push***.
+```javascript
+ events.push("KLOS Golden Gala","An evening with hollywood vampires");
```

Comment:
1. Be sure an array exists to push to. 
2. You are not pushing event objects right now, you are pushing strings.